portal/src/main/webapp/js/src/oncokb/OncoKBConnector.js

```diff
@@ -157,6 +157,7 @@ var OncoKB = (function(_, $) {
     function Instance(id) {
         this.initialized = false;
         this.dataReady = false;
+        this.civicDataReady = false;
```

Comment:
CIViC does not rely on data coming from OncoKB, but we did it inside OncoKBConnector.js to get advantage of mainly the OncoKB event handlers without having to duplicate the code.